### ISSUE TITLE
Broken PECL archive

### ISSUE THREAD

**Issue by remicollet** [awslabs/aws-crt-php#34]
Probably because missing sources for used libraries...
```

$ make
cmake3 -DCMAKE_INSTALL_PREFIX=build/install -DBUILD_TESTING=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo -Hcrt/aws-crt-ffi -Bbuild/aws-crt-ffi-static -DBUILD_SHARED_LIBS=OFF
CMake Error: The source directory "/tmp/awscrt-1.0.0/crt/aws-crt-ffi" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.

```

P.S. build from a git recursive clone works, so really missing file in the PECL archive

**Comment by remicollet** [comment]
Also package.xml is not tracked

And lot of files have bad role (doc where src is expected)

**Comment by cmb69** [comment]
This is actually the same issue as #33.

**Comment by remicollet** [comment]
closing as duplicate of #33 and still not fixed,  issue #37
